features/bicycle/cycleway.feature

```diff
@@ -85,13 +85,21 @@ Feature: Bike - Cycle tracks/lanes
             | highway     | cycleway | oneway | forw    | backw        |
             | motorway    | track    | yes    | cycling |              |
             | residential | track    | yes    | cycling | pushing bike |
-            | cycleway    | track    | yes    | cycling | pushing bike |
+            | cycleway    | track    | yes    | cycling |              |
```

Comment:
i don't think a oneway cycleway should allow pushing bikes in the reveres direction